packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -99,14 +97,10 @@ export function BlockSettingsDropdown( { clientIds, ...props } ) {
 									text={ () => serialize( blocks ) }
 									role="menuitem"
 									className="components-menu-item__button"
-									onCopy={ () => {
-										setHasCopied( true );
-									} }
-									onFinishCopy={ () => setHasCopied( false ) }
+									onCopy={ onCopy }
+									onFinishCopy={ onClose }
```

Comment:
I'm not sure if I expect it to close after a delay. Either we should close it immediately or not close at all.

Reply:
You are right. I believe we should probably leave it open. I'll change it. Thanks!